pandas/core/arrays/datetimes.py

```diff
@@ -1007,10 +1007,39 @@ def date(self):
         'dim',
         "The number of days in the month")
     daysinmonth = days_in_month
+    _is_month_start_doc = """
+        Indicates whether the date is the first day of the month.
+
+        Returns
+        -------
+        Series of bool
+            Series of boolean values indicating whether the date is the first
+            day of the month.
```

Comment:
This generally looks good, only issue is that it isn't always a Series.  On DatetimeIndex it returns a numpy array.

Reply:
True. @CalmDownKarm you can take a look at `is_month_end` where this is correct. And as I said, if you want to merge both docstrings into one that would be great. It'd be something like:
```
is_month_doc = """
Indicates whether the date is the {first_or_last} day of the month.
[...]
"""
is_month_start = _field_accessor('is_month_start', 'is_month_start', is_month_doc.format(first_or_last='first'))
```

Reply:
fixed and combined. @datapythonista @jbrockmendel